recipes/selinux/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "selinux" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/selinux-{{ version }}.tar.gz
+  sha256: d435f514e834e3fdc0941f6a29d086b80b2ea51b28112aee6254bd104ee42a74
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```

I think this can be made noarch even though it only makes sense on linux (it's fine for the CI to fail on macOS and Windows as a result):
```suggestion
  noarch: python
```